app/assets/stylesheets/home.scss

```diff
@@ -16,4 +16,21 @@
   padding: 2px;
   color: red;
   display: table;
-}
\ No newline at end of file
+}
+
+div.random_card_for_check {
```

Comment:
Properties should be ordered background-color, border, border-radius, margin-top, padding-bottom